x-pack/plugins/apm/public/hooks/use_filters_for_embeddable_charts.ts

```diff
@@ -8,14 +8,17 @@
 import { useMemo } from 'react';
 import { SERVICE_NAME } from '../../common/es_fields/apm';
 import { termQuery } from '../../common/utils/term_query';
-import { useApmParams } from './use_apm_params';
+import { useAnyOfApmParams } from './use_apm_params';
 import { environmentQuery } from '../../common/utils/environment_query';
 
 export function useFiltersForEmbeddableCharts() {
   const {
     path: { serviceName },
     query: { environment },
-  } = useApmParams('/mobile-services/{serviceName}/overview');
+  } = useAnyOfApmParams(
+    '/mobile-services/{serviceName}/overview',
+    '/mobile-services/{serviceName}/transactions'
+  );
```

Comment:
Same here, can't we receive `serviceName` and `environment` as arguments of this hook instead of using  `useAnyOfApmParams`?